receiver/zookeeperreceiver/README.md

```diff
@@ -0,0 +1,16 @@
+**Under development - not ready for use.**
+
+# Zookeeper Receiver
+
+The Zookeeper receiver collects metrics from a Zookeeper instance, using the `mntr` command.
```

Comment:
In versions starting 3.6.0, Zookeeper exposes metrics via prometheus. However, this is a recent release (March 2020) and it appears to be able to collect metrics from previous versions (starting 3.4.0) `mntr` command is the way to go. I think to begin with we can start syncing metrics from `mntr` and once the prometheus style metrics have been around for long enough, we can update the receiver appropriately. FYI @keitwb

zk release notes: https://zookeeper.apache.org/releases.html
tickets discussing prometheus metrics provider: https://issues.apache.org/jira/browse/ZOOKEEPER-3091, https://issues.apache.org/jira/browse/ZOOKEEPER-3092

Reply:
Well, we won't even need a special receiver if we are gathering prometheus metrics, we would just point people to the prometheus receiver.